cfme/tests/intelligence/chargeback/test_resource_allocation.py

```diff
@@ -401,16 +401,15 @@ def generic_test_chargeback_cost(chargeback_costs_custom, chargeback_report_cust
         3.Validate the costs reported in the chargeback report.The costs in the report should
           be approximately equal to the cost estimated in the resource_cost fixture.
     """
-    for groups in chargeback_report_custom:
-        if not groups[column]:
-            pytest.skip('missing column in report')
-        else:
-            estimated_resource_alloc_cost = chargeback_costs_custom[resource_alloc_cost]
-            cost_from_report = groups[column]
-            cost = cost_from_report.replace('$', '').replace(',', '')
-            soft_assert(estimated_resource_alloc_cost - COST_DEVIATION <=
-                float(cost) <= estimated_resource_alloc_cost + COST_DEVIATION,
-                'Estimated cost and report cost do not match')
+    if not chargeback_report_custom[0][column]:
+        pytest.skip('missing column in report')
+    else:
+        estimated_resource_alloc_cost = chargeback_costs_custom[resource_alloc_cost]
+        cost_from_report = chargeback_report_custom[0][column]
```

Comment:
**Required** Lets add a comment here on why we are taking index 0. Same comment below